README.md

```diff
@@ -18,9 +20,10 @@ Personal note taking app that enables users to share their notes with other user
 
 Video and tutorial on how our project works.
 
-## Documentation
+## Documentation  
```

Comment:
Can we remove these changes here since we're deploying documentation through the main page? Otherwise looks good to me!